cluster/src/main/java/com/hartwig/pipeline/input/MetadataProvider.java

```diff
@@ -33,6 +33,7 @@ public SomaticRunMetadata get() {
                         .barcode(barcodeOrSampleName(t))
                         .sampleName(t.name())
                         .primaryTumorDoids(t.primaryTumorDoids())
+                        .samplingDate(t.samplingDate())
```

Comment:
What's the process for populating the SampleInput? 

I assume `samplingDate` is an optional field also in the SampleInput?

Reply:
It is the responsibility of the launcher of the pipeline to pass it in as config (same with the doids). In the end it comes from LAMA, same field as is currently extracted in the actin script.